test/e2e/framework/framework.go

```diff
@@ -593,8 +593,8 @@ func KubeDescribe(text string, body func()) bool {
 }
 
 // ConformanceIt is wrapper function for ginkgo It.  Adds "[Conformance]" tag and makes static analysis easier.
-func ConformanceIt(text string, body interface{}, timeout ...float64) bool {
-	return ginkgo.It(text+" [Conformance]", body, timeout...)
+func ConformanceIt(profile, text string, body interface{}, timeout ...float64) bool {
+	return ginkgo.It(text+" [Conformance] [Profile:" + profile +"]", body, timeout...)
```

Comment:
while the format seems fine, the conformance testing KEP or the conformance profiles KEP do not set the format in stone (or at least i could not find it).

does it make sense to do that?